home_work_1_zd_3.js

```diff
@@ -0,0 +1,24 @@
+const MASSNUMS = [4, 7, 9, 2, 8, 4, 10, 15, 6, 19];
+// const MASSNUMS = [-1, 0, 1, 2, 4, 5];
+let flag = true;
+
+function get_avg(mass) {
```

Comment:
please follow documented naming for the JS functions

Reply:
Done